primer/brand
# Add outline for `Button` in HCM

## Summary

Adds border outline to `Primary` and `Secondary` Button variants in High Contrast Mode (HCM). This fix is tied to [#1683](https://github.com/github/primer/issues/1683). 

## List of notable changes:



- Adds CSS styles specific to HCM to display a border around specific Button variants.



## Steps to test:



* On Windows, turn on High Contrast Mode to see changes.
* On Mac/Chrome, turn on "Force-colors" rendering setting under "Emulate CSS Media feature forced-colors" to see an emulated version of HCM.

## Supporting resources (related issues, external links, etc):

- [Chrome HCM emulation](https://developer.chrome.com/docs/devtools/rendering/emulate-css/#emulate-css-media-feature-forced-colors)

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">



<img width="200" alt="Button 'Primary' variant, shows a button with no outline in HCM" src="https://user-images.githubusercontent.com/26746305/224075381-d2cb2f80-9dc2-43e7-b7fa-b0e116c9845b.png">

 </td>
<td valign="top">



<img width="205" alt="Button 'Primary' variant, shows a button with a yellow outline in HCM" src="https://user-images.githubusercontent.com/26746305/224077824-6a87aef5-96ee-40df-a50c-7dbb0ac5a25b.png">

</td>
</tr>
<tr>

<td>

<img width="260" alt="Button 'Primary' variant, shows a disabled button with no outline in HCM" src="https://user-images.githubusercontent.com/26746305/224076271-802b54a0-c924-439b-beb2-1e2a637e7e47.png">

</td>

<td>

<img width="276" alt="Button 'Primary' variant, shows a disabled button with a green outline in HCM" src="https://user-images.githubusercontent.com/26746305/224076547-1ffbd080-c18e-4f36-898e-3b9ea01a3c1d.png">

</td>

</tr>
</table>
